packages/network/src/api/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { THUMBNAIL_RESIZE_RATIO, Image } from '@monkvision/types';
```

Comment:
Why is the `THUMBNAIL_RESIZE_RATIO` constant declared in the types package by the way? If it is only used by the network package, I think we should put it in the network package.